app/ts/simulation/services/SimulationModeEthereumClientService.ts

```diff
@@ -346,9 +346,12 @@ export const getSimulatedCode = async (ethereumClientService: EthereumClientServ
 		accessList: []
 	} as const
 	const multiCall = await simulatedMulticall(ethereumClientService, simulationState, [getCodeTransaction], blockNum + 1n)
+	const lastResult = multiCall[multiCall.length - 1]
```

Comment:
you can also a modern builtin `at` [msdn](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/at)
```suggestion
	const lastResult = multiCall.at(- 1)
```